src/css/custom.css

```
@@ -1,3 +1,6 @@
+@import url("https://use.typekit.net/ymm4lqb.css");
```

Stylelint (17.4.0)

[error] Expected "url("https://use.typekit.net/ymm4lqb.css")" to be ""https://use.typekit.net/ymm4lqb.css"" (import-notation)

(import-notation)